src/Encoding/FilteredStream.php

```diff
@@ -23,21 +23,6 @@
     protected $readFilterCallback;
 
     /**
-     * @var resource
-     */
-    protected $readFilter;
-
-    /**
-     * @var callable
-     */
-    protected $writeFilterCallback;
```

Comment:
In that case, the `$readFilter` and `$writeFilter` properties should remain and should be marked as deprecated too.